### ISSUE TITLE
cookie not getting read correctly from jest test in 4.0.2

### ISSUE THREAD

**Issue by jonathanhculver** [reactivestack/cookies#237]
Hello!

Opening in case other's have had the same issue. In `4.0.2` I have a jest test that set's a cookie and a function being tested that read's the cookie. The cookie comes back as `undefined`. 

Downgrading to `4.0.0` fixes the issue for me.

#### Example:

```js
// testFunc.js
import Cookies from 'universal-cookie';

const cookies = new Cookies();
const testFunc = () => (cookies.get('testCookie'));

// testFunc.spec.js
import Cookies from 'universal-cookie'
import testFunc from './testFunc';

const cookies = new Cookies();

test('testFunc', () => {
  cookies.set('testCookie', 'blah');
  // logging cookies.get('testCookie') correctly logs 'blah'

  const result = testFunc();
  expect(result).toEqual('blah'); // fails - result is actually `undefined`
});

**Comment by JHughes11** [comment]
Hello @jonathanhculver! I'm having the same issue that you're having. Looking through the source code, I believe I found the root of the problem. 

Seen [here](https://github.com/reactivestack/cookies/compare/v4.0.0...v4.0.1?diff=split#diff-5eb3b3faec37015291bb92e4845b553a)

the `DOCUMENT_HAS_COOKIE` is initialized as false in versions greater than 4.0.0. 

Setting this value to `true` in both the `.js` file and `test` file seems to fix this issue. But I would just stick with the 4.0.0 version until it can get fixed :D

**Comment by eXon** [comment]
Fixed with `v4.0.3`

**Comment by bhushan-patil-official** [comment]
For me I think while testing I had to wrap the component with CookieProvider
